tools/get_platform.py

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env python
-import pkgutils
+"""Module to Get platform information """
```

Comment:
Anyone reading it knows that it's a module, and the name of the module is `get_platform`. Your comment mostly just repeats that same information. That's a hallmark of a bad comment. A good comment would explain what the module is supposed to do and possibly give some usage examples.